lib/ansible/playbook/base.py

```diff
@@ -778,7 +778,7 @@ def get_search_path(self):
         Return the list of paths you should search for files, in order.
         This follows role/playbook dependency chain.
         '''
-        path_stack = []
+        path_stack = [] + C.LOCAL_SEARCH_PATH
```

Comment:
this is not the place you should add this as it influences a lot more than the search path for copy. See the 'dwim' functions.